app/helpers/application_helper.rb

```diff
@@ -28,9 +28,9 @@ def background_image_style(img)
     "background-image: url(\"#{img}\");"
   end
 
-  def react_component(id:, component:, props: {}, wrapper_tag: 'div')
+  def react_component(id:, component:, props: {}, wrapper_tag: 'div', classes: '')
     camel_props = props.deep_transform_keys { |key| key.to_s.camelize(:lower) }
-    tag.send(wrapper_tag, { id: id, class: REACT_COMPONENT_CLASS, data: { component: component, react_props: camel_props } }) {}
+    tag.send(wrapper_tag, { id: id, class: class_names(REACT_COMPONENT_CLASS, classes), data: { component: component, react_props: camel_props } }) {}
```

Comment:
when we're in flexy things, we may need to add classes to the wrapper div.  This allows for that.